docs/reference-guide/query-modules/implement-custom-query-modules/custom-query-module-example.md

```diff
@@ -213,6 +213,150 @@ def write_procedure(context: mgp.ProcCtx,
     return mgp.Record(created_vertex=vertex)
 
 ```
+### Batched read procedures
+
+Similar to regular `read` and `write` procedures, we have `batched` `read` and `write` procedures. Batched procedures are very similar to regular procedures. The key difference is that batched procedures return results in batches, mostly to reduce memory consumption. For batched procedures, you need to define **three** functions:
+* `batching` function - similar to the main function in regular procedures
+* `initialization` function - function to initialize stream, open source file, etc...
+* `cleanup` function - function to close a stream, source file, etc...
+
+Since there are three functions, construct works as follows: 
+- `initialization` function must define to receive the same parameters in the same order as batching function including `mgp.ProcCtx` if defined as the first parameter
+- when calling procedure from the query, you need to call the `batching` function
+- Memgraph calls `initialization` before the `batching` function 
+- `batching` function needs to return an empty result at some point, which signals the end of the stream
+- `cleanup` function is called at end of the stream
```

Comment:
```suggestion
- `initialization` function must be defined in a way it receives the same parameters in the same order as batching function, including `mgp.ProcCtx` if it's defined as the first parameter
- when calling the procedure from the query, you need to call the `batching` function
- Memgraph calls `initialization` before the `batching` function 
- `batching` function needs to return an empty result at some point, which signals the end of the stream
- `cleanup` function is called at the end of the stream
```

Reply:
if it's ok fix the same thing in the write procedures chapter

Reply:
Done